projects/ui/src/lib/components/po-field/po-combo/samples/sample-po-combo-heroes-reactive-form/sample-po-combo-heroes-reactive-form.component.ts

```diff
@@ -21,6 +21,10 @@ export class SamplePoComboHeroesReactiveFormComponent implements OnInit {
     });
   }
 
+  isHeroSelected() {
+    return this.form.valid ? 'Know more' : undefined;
```

Comment:
o termo is..., ex: isNumber..
é comum utilizar para retornar um booleano..

Nesse caso esta retornando uma string ou undefined..

Sugestão:
- knowMoreLabel
- primaryLabel

Aproveitando daria pra criar um get e evitar função na view.

Obs: também verificar o outro sample.